packages/react-scripts/template/README.md

```diff
@@ -430,8 +429,8 @@ Then we can change `start` and `build` scripts to include the CSS preprocessor c
 
 ```diff
    "scripts": {
-     "build-css": "node-sass src/ -o src/",
-     "watch-css": "npm run build-css && node-sass src/ -o src/ --watch --recursive",
+     "build-css": "node-sass-chokidar src/ -o src/",
+     "watch-css": "npm run build-css && node-sass-chokidar src/ -o src/ --watch --recursive",
```

Comment:
without the `build-css` inside of `watch-css` there is no guarantee on the latest stylesheets being used..

so,
 1. If it's the first time running `npm run start` you won't have ANY CSS files yet
 1. If you made changes to a stylesheet while you weren't watching then `npm run start` those changes won't show until you modify the stylesheet again